charts/kyuubi/values.yaml

```diff
@@ -45,7 +45,7 @@ updateStrategy:
 image:
   repository: apache/kyuubi
   pullPolicy: IfNotPresent
-  tag: ~
+  tag: 1.8.1-spark
```

Comment:
use an image that contains Spark binary as the default value.

Reply:
I would prefer to leave `~` for now to use `{{ .Chart.AppVersion }}` as image tag value by default.
The change makes change, however it's not enough to deploy Kyuubi Helm chart with Spark support.
Let me create `Minimal Helm chart configuration to run Kyuubi with Spark` issue to discuss all problems, limitations and options there.

Reply:
reverted